wgpu/gui/base.py

```diff
@@ -41,6 +42,21 @@ def log_exception(kind):
             logger.error(msg + f" ({count})")
 
 
+def weakbind(method):
+    """Wrap a method so that the instance is held in a weakref."""
```

Comment:
More precise docstring.
```suggestion
    """Replace a bound method with a callable object that stores the `self` using a weakref."""
```